packages/fxa-payments-server/src/lib/apiClient.test.ts

```diff
@@ -1,71 +1,69 @@
+import { CouponDetails } from 'fxa-shared/dto/auth/payments/coupon';
```

Comment:
These `import` order changes are related to a small refactor to move `deepCopy` to a utility file and import it in various places it's used. Since I added an `import` statement to the relevant files, I also cleaned up the imports via the TypeScript Hero VS Code extension.